test/InterchainTokenService.js

```diff
@@ -1510,86 +1433,12 @@ describe('Interchain Token Service', () => {
         });
     });
 
-    describe('Execute with token checks', () => {
-        const sourceChain = 'source chain';
-        let sourceAddress;
-        const amount = 1234;
-        let destAddress;
-        const tokenName = 'Token Name';
-        const tokenSymbol = 'TS';
-        const tokenDecimals = 16;
-
-        before(async () => {
-            sourceAddress = service.address;
-            destAddress = wallet.address;
-            await deployFunctions.gateway(service, tokenName, tokenSymbol, tokenDecimals);
-        });
-
-        it('Should revert on execute with token if remote address validation fails', async () => {
-            const commandId = await approveContractCallWithMint(
-                gateway,
-                sourceChain,
-                wallet.address,
-                service.address,
-                '0x',
-                tokenSymbol,
-                amount,
-            );
-
+    describe('Execute with token', () => {
+        it('should revert on execute with token', async () => {
             await expectRevert(
-                (gasOptions) => service.executeWithToken(commandId, sourceChain, wallet.address, '0x', tokenSymbol, amount, gasOptions),
+                (gasOptions) => service.executeWithToken(HashZero, 'sourceChain', wallet.address, '0x', 'TEST', 1, gasOptions),
```

Comment:
Does `commandId` param needs to be changed to `HashZero`?
```suggestion
                (gasOptions) => service.executeWithToken(commandId, 'sourceChain', wallet.address, '0x', 'TEST', 1, gasOptions),
```

Reply:
Yes this is just checking that this method reverts